### PR TITLE
Add reviewer guidelines

### LINKED ISSUE #131: Need Review Guidelines for reviewers

### PR DESCRIPTION
**Pull request by rbeucher**
Fix #131

### CODE REVIEW

File: docs/about/contribute/reviewers_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+A review involves checking submissions against a checklist of essential features and details described at the top of each PR.
+This should be objective, not subjective; it should be based on the materials in the submission as perceived without distortion by personal feelings, prejudices, or interpretations.
+
+Some continuous tests such as hyperlink references checks and preview deployments are automatically triggered by submitting a PR.
+
+Reviewers should:
+
+1. Ensure that the tests are passing without errors.
+1. Do a visual check using the preview.
+1. Do a Github pull request review [See extensive GitHub documentation][github-review]
```
Comment (ccarouge) [inline docs/about/contribute/reviewers_guidelines.md:25]:
```suggestion
1. Do a Github pull request review. [See GitHub's extensive documentation][github-review]
```

File: docs/about/contribute/reviewers_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+1. Once you have approved the PR. Tag the **editors team** `@ACCESS-Hive/editors` in the discussion.
+
+We encourage reviewers to provide feedback from within the PR discussion.
+
+You can include in your review links to any new issues that you the reviewer believe to be impeding the acceptance of the repository.
```
Comment (ccarouge) [inline docs/about/contribute/reviewers_guidelines.md:30]:
```suggestion
You can include in your review links to any new issues that you the reviewer believe to be impeding the acceptance of the pull request.
```

File: docs/about/contribute/reviewers_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Reviewing for ACCESS-Hive
+
+Any member of the [ACCESS-Hive Github Community][access-hive-github] can join the reviewers team.
+Please ask one of the [maintainers][access-hive-maintainers] to invite you to join the organisation.
```
Comment (ccarouge) [inline docs/about/contribute/reviewers_guidelines.md:4]: In the contribution guidelines, we say to go through the forum. Should we change to the maintainers team? It probably makes more sense. What do you think is the best way to contact the maintainers?

**Review by ccarouge — CHANGES_REQUESTED** [review]
There are some minor changes. And a discussion to resolve before merging.

File: docs/about/contribute/reviewers_guidelines.md (hunk 3 above)
Comment (rbeucher) [reply docs/about/contribute/reviewers_guidelines.md:4]: I think in that context maintainers is better. I'll change the contribution guidelines

**Review by ccarouge — APPROVED** [review]

**Pull request merged**